### PR TITLE
Implement local submitter for TO RUN in SQLFlow runtime.

### PR DESCRIPTION
**Pull request by brightcoder01**

### CODE REVIEW

File: python/runtime/local/submitter.py [hunk 1]
```diff
@@ -165,13 +169,50 @@ def submit_local_explain(datasource,
...
+        args.extend(params[1:])
+        sub_process = subprocess.run(args=args,
+                                     env=subprocess_env,
+                                     stdout=subprocess.PIPE,
+                                     stderr=subprocess.PIPE)
+    else:
+        print(
+            "The other executable except Python program is not supported yet")
+
+    if sub_process:
```
Comment (sneaxiy) [inline python/runtime/local/submitter.py:214]: How about check `sub_process.exitcode == 0` to find whether the subprocess runs well? If `sub_process.exitcode != 0`, try to print `sub_process.stderr` to the stderr of the parent process.

**Comment by sneaxiy** [comment]
Try to enable the `e2e_workflow_test.go` for `TO RUN` statements as well.

https://github.com/sql-machine-learning/sqlflow/blob/9bcbe31872569c87fcb666f9b628988084287e04/go/cmd/sqlflowserver/e2e_workflow_test.go#L84-L102

File: python/runtime/local/submitter.py (hunk 1 above)
Comment (brightcoder01) [reply python/runtime/local/submitter.py:214]: Good idea, updated.

File: python/runtime/local/submitter.py [hunk 2]
```diff
@@ -165,13 +170,51 @@ def submit_local_explain(datasource,
...
+                                     stderr=subprocess.PIPE)
+    elif file_ext.lower() == ".py":
+        args = ["python", "-m", program_file_path.stem]
+        args.extend(params[1:])
+        sub_process = subprocess.run(args=args,
+                                     env=subprocess_env,
+                                     stdout=subprocess.PIPE,
+                                     stderr=subprocess.PIPE)
+    else:
+        print(
```
Comment (sneaxiy) [inline python/runtime/local/submitter.py:211]: Should we raise an error here instead of printing a message?

Comment (brightcoder01) [reply python/runtime/local/submitter.py:211]: Good idea, updated.

**Review by sneaxiy — APPROVED** [review]

**Pull request merged**